packages/site/src/layout/SearchBox.tsx

```diff
@@ -126,6 +129,39 @@ export const SearchBox = ({
                 </ContentCardWrapper>
               </Content>
             )}
+            {filteredComponentList.length == 0 &&
+              filteredDesignList.length == 0 && (
+                <Box
+                  height={"100%"}
+                  direction={"column"}
+                  padding={"extravagant"}
+                  gap={"larger"}
+                  alignItems={"center"}
+                >
+                  <ToolBoxIllustration />
+                  <Heading level={1} element={"h3"}>
+                    The toolbox looks empty!
+                  </Heading>
+                  <Typography
+                    align={"center"}
+                    fontWeight={"semiBold"}
+                    size={"large"}
+                    textColor={"text"}
+                  >
+                    We couldn&apos;t match any results with your search; try a
+                    different term.
+                  </Typography>
+                  <Typography
+                    align={"center"}
+                    fontWeight={"medium"}
+                    size={"large"}
+                    textColor={"textSecondary"}
+                  >
+                    If you think something&apos;s missing, let the Atlantis team
+                    know.
```

Comment:
Copy is a bit off the cuff but trying to follow our [general guidance on errors](https://atlantis.getjobber.com/storybook/?path=/docs/content-voice-tone--docs#error-messages-should-be-helpful)

Reply:
Another idea could be linking out to the Components/Design/Content/etc pages, or even just Home?

Reply:
I think its good! It could be fun to link to a random page or something.